appnav/src/main/kotlin/io/element/android/appnav/LoggedInFlowNode.kt

```diff
@@ -238,6 +239,7 @@ class LoggedInFlowNode @AssistedInject constructor(
                         }
                     }
                 } else {
+                    analyticsService.capture(ViewRoom(isDM = room.isDirect))
```

Comment:
Perhaps we should put tracking ode in the node that it directly relates with (i.e. not in the parent node).
What if this was in `RoomFlowNode.init{}` ?